Development/src/pages/logs.js

```diff
@@ -1,92 +1,81 @@
 import React, { useState } from 'react';
-import {
```

Comment:
This file needs splitting out into LogsShow.js and LogsList.js like the others, right?